src/components/ScopedNotification/ScopedNotification.js

```diff
@@ -4,10 +4,21 @@ import cx from 'classnames';
 import { Icon } from '../Icon';
 import { MediaObject } from '../MediaObject';
 
+const defaultIconRenderer = (theme, size) => (
```

Comment:
Maybe better to pass params ar an object? `{ theme, size }`
This way You can just pass function components to `renderIcon`

Reply:
Yes good point, will change